<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEditAnv</class>
 <widget class="QDialog" name="dlgEditAnv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiPAR - Edição de Aeronaves</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="11" column="4" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblKey">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sigla:</string>
     </property>
     <property name="buddy">
      <cstring>qleKey</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="qsbRazMaxDesc">
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="specialValueText">
      <string>Unknown</string>
     </property>
     <property name="suffix">
      <string> ft/min</string>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="lblRazMaxSub">
     <property name="text">
      <string>Razão Máxima de Subida:</string>
     </property>
     <property name="buddy">
      <cstring>qsbRazMaxSub</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="qsbVelApx">
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="specialValueText">
      <string>Unknown</string>
     </property>
     <property name="suffix">
      <string> kt</string>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>130</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblVelApx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Velocidade de Aproximação:</string>
     </property>
     <property name="buddy">
      <cstring>qsbVelApx</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblRazMaxDesc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Razão Máxima de Descida:</string>
     </property>
     <property name="buddy">
      <cstring>qsbRazMaxDesc</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblDescr">
     <property name="text">
      <string>Descrição:</string>
     </property>
     <property name="buddy">
      <cstring>qleDescr</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="qleDescr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="qleKey">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QSpinBox" name="qsbRazMaxSub">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> ft/min</string>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>2800</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="lblRazMaxCurv">
     <property name="text">
      <string>Razão Máxima de Curva:</string>
     </property>
     <property name="buddy">
      <cstring>qsbRazMaxCurv</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QSpinBox" name="qsbRazMaxCurv">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> gr/seg</string>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblTetoServ">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Teto de Serviço:</string>
     </property>
     <property name="buddy">
      <cstring>qsbTetoServ</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSpinBox" name="qsbTetoServ">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> ft</string>
     </property>
     <property name="minimum">
      <number>15000</number>
     </property>
     <property name="maximum">
      <number>90000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>40000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qleKey</tabstop>
  <tabstop>qleDescr</tabstop>
  <tabstop>qsbVelApx</tabstop>
  <tabstop>qsbRazMaxSub</tabstop>
  <tabstop>qsbRazMaxDesc</tabstop>
  <tabstop>qsbRazMaxCurv</tabstop>
  <tabstop>qsbTetoServ</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgEditAnv</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgEditAnv</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
